src/messages/GE.json

```diff
@@ -280,5 +280,13 @@
   "running the query": "When you sign in, running the query will affect the data in your tenant.",
   "We did not find any history items": "We did not find any history items",
   "Expand response": "Expand response",
-  "Access Token": "Access token"
+  "Access Token": "Access token",
+  "consent to scopes": "Go to the Modify permissions tab and consent to scopes",
```

Comment:
```suggestion
  "consent to scopes": "You need to consent to the permissions on the Modify permissions tab.",
```

Reply:
I have changed this to fit in with the bold styling